docs/workloads/azure/data/getting_started/core_data_platform_deployment_azure.md

```diff
@@ -7,12 +7,94 @@ hide_table_of_contents: false
+```
+
+Open the project locally and push the generated project to the target remote repository's `main` branch.
+
+## Step 2: Add Infrastructure pipeline in Azure DevOps
+
+Open the project locally and create a new feature branch e.g.:
+
+```bash
+git checkout -b feat/infra-pipeline
+```
+
+YAML file containing a template Azure DevOps CI/CD pipeline for building and deploying the core infrastructure, named `air-infrastructure-data.yml` is configured in the `build/azDevOps/azurefolder`
+
+YAML file `air-infrastructure-data.yml` should be added as the definition for a new pipeline in Azure DevOps.
+
+1. Sign-in to your Azure DevOps organization and go to your project
+2. Go to Pipelines, and then select **New pipeline**
+3. Name the new pipeline, e.g. `amido.stacks-data-infrastructure`
+4. For the pipeline definition, specify the YAML file in the project repository feature branch (`air-infrastructure-data.yml`) and save
+5. The new pipeline will require access to any Azure DevOps pipeline variable groups specified in the pipeline YMAL. Under each variable group, go to 'Pipeline permissions' and add the pipeline.
+
+The `build/azDevOps/azure/network` folder contains a YAML file named `air-infrastructure-data-network.yml` that serves as a template for Azure DevOps CI/CD pipeline for networking and self-hosted agent infrastructure. If the client or user does not have their own network and self-hosted agent, they must establish the networking pipeline in Azure DevOps following the procedures outlined above. This setup should be completed before initiating the core infrastructure pipeline. The `build/azDevOps/azure/network` folder also includes another YAML file called `air-infrastructure-data-network-vars.yml` that contains the variables used in the networking pipeline. These variables must be updated as well as the project requirements.
+
+If the client/user has their own network and self-hosted agent, then the networking pipeline won't be required.
+
+## Step 3: Deploy Infrastructure in non-production environment
+
+Run the pipeline configured in Step 2 to commence the build and deployment process.
+
+Running this pipeline in Azure DevOps will initiate the deployment of artifacts into the non-production (nonprod) environment.It's important to monitor the progress of this deployment to ensure its success. You can track the progress and status of the deployment within the Pipelines section of Azure DevOps.
+
+If successful, the core infrasturcture resources will now be available in the nonprod Stacks environment. To view these deployed resources, navigate to the Azure portal and search for the resource group associated with the deployment. This resource group is named based values provided during step 1, such as company name, project name,region,component. Within the resource group, you'll find a list of the resources that were deployed.
+
+Name Format for Resource Groups  - companyname-projectname-stage-region-component eg amido-stacks-dev-euw-de
```

Comment:
Can remove this if using the suggestion above.

Reply:
updated